index.js

```diff
@@ -47,6 +47,15 @@ exports.getMeasuresSchema = function(performanceYear = 2017) {
   return YAML.load(path.join(__dirname, 'measures', performanceYear.toString(), 'measures-schema.yaml'));
 };
 
+
+/**
+ * @return {Array<Measure>}
+ */
+exports.getMeasuresDataByPerfYear = function(performanceYear) {
```

Comment:
This function is exactly the same as `exports.getMeasuresData = function(performanceYear = 2017) {` on line 38 and as such is unnecessary.